view/text_widget.go

```diff
@@ -13,9 +13,9 @@ type TextWidget struct {
 }
 
 // NewTextWidget creates and returns an instance of TextWidget
-func NewTextWidget(app *tview.Application, commonSettings *cfg.Common, focusable bool) TextWidget {
```

Comment:
TODO: Get rid of passing `defaultFocusable` into the constructor. Have that be a constant in the module. Use the config value first and default to the module constant.